edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -67,6 +70,15 @@ def authenticate(self, request):
             if not user_and_auth:
                 return user_and_auth
 
+            # Fail authentication if user disabled
+            user = user_and_auth[0]
+            if user and isinstance(user, get_user_model()):
+                if not user.has_usable_password():
+                    msg = 'User disabled by admin: %s' % user.get_username()
+                    raise exceptions.AuthenticationFailed({
```

Comment:
can we please follow the error msg standard already present in JWT implementation?

Reply:
@jawad-khan Can you please specify? In the current file I do not see any message being passed in `AuthenticationFailed()`.  
On the other hand, the current message added is in line with the message passed in bearer token authentication in the same case (User disabled by admin).